spec/syntax.md

```diff
@@ -50,6 +51,17 @@ as the *unprefixed identifier*.
 
 No whitespace is allowed between components of an `InterpolatedIdentifier`.
 
+### `InterpolatedDynamicUrl`
```

Comment:
I'm not sure what "Dynamic" means here. Is it intended to disambiguate an interpolated URL that can't contain a SassScript expression from one that can?

Reply:
well, `dynamicUrl` is the way it is called in the dart-sass source code.

Reply:
That's for implementation-level reasons (specifically, to disambiguate it from the way we parse "static" URLs in plain-CSS stylesheets). I think just saying `InterpolatedUrl` here is fine.